db/data.py

```diff
@@ -409,11 +409,46 @@ def load_high_level(mbid, offset=0):
             data = row[1]
             version = row[2]
             data["version"] = version
+            if map_keys:
+                data = map_model_classes(model,data)
```

Comment:
I think that this would be better if we separated the logic for getting the highlevel data with the logic for mapping the class names. This means I think we should leave this method as it used to be, and have another method to load the mapping from the database. This could return: `{'model_name': {the mapping}, 'another model': {mapping}`, etc.
Then, in each highlevel API endpoint, if the parameter is set, perform a processing step that converts these values. This will make it easier to test that we can perform the mapping.